.github/actions/download-rpm/action.yaml

```
@@ -9,3 +9,4 @@ runs:
           VERTICA_CE_URL: "https://vertica-community-edition-for-testing.s3.amazonaws.com/XCz9cp7m/vertica-12.0.3-0.x86_64.RHEL6.rpm"
       run: |
         curl $VERTICA_CE_URL -o docker-vertica/packages/vertica-x86_64.RHEL6.latest.rpm
+        curl $VERTICA_CE_URL -o docker-vertica-v2/packages/vertica-x86_64.RHEL6.latest.rpm
```

I was hoping we could have a parameter that picks the location. But an alternative is to just do a copy. Then we save having to download the file twice.
```suggestion
        cp docker-vertica/packages/*rpm docker-vertica-v2/packages
```